cli/src/formatter.rs

```diff
@@ -161,6 +161,7 @@ impl FormatterFactory {
     }
 
     pub fn color(config: &StackedConfig, debug: bool) -> Result<Self, ConfigGetError> {
+        crossterm::style::force_color_output(true);
```

Comment:
nit: Since this is a global flag, it might be better to do this in `CliRunner` or somewhere closer to `main()`.

Reply:
Yeah, I wasn't sure where the best place would be. I moved it to `CliRunner::run()`. PTAL.